### PR TITLE
Shorten ru.tinkoff.tschema to tschema, tidy up modules structure a bit

### PR DESCRIPTION
**Pull request by REDNBLACK**

### CODE REVIEW

**Comment by REDNBLACK** [comment]
> java.util.MissingResourceException: Can't find bundle for base name org.scalatest.ScalaTestBundle, locale en_US

😠

File: build.sbt [hunk 1]
```diff
@@ -213,8 +216,8 @@ lazy val swagger = project
     circe
   )
 
-lazy val akkaHttp = project
-  .in(file("modules/akkaHttp"))
+lazy val `akka-http` = project
```
Comment (Odomontois) [inline build.sbt]: I don't see this as an improvement.
My practice shows that backticked names are cumbersome in development somehow.

File: modules/akkaHttp/src/main/scala/tschema/akkaHttp/FindKey.scala [hunk 2]
```diff
@@ -1,5 +1,5 @@
-package ru.tinkoff.tschema.akkaHttp
-import ru.tinkoff.tschema.common.Find
+package tschema.akkaHttp
```
Comment (Odomontois) [inline modules/akkaHttp/src/main/scala/tschema/akkaHttp/FindKey.scala]: We have to add deprecated aliases for old names.

**Comment by danslapman** [comment]
I don't think we should do this at all. Renaming packages will simply add one more trouble for end-users, but what we will gain?

File: modules/akkaHttp/src/main/scala/tschema/akkaHttp/FindKey.scala (hunk 2 above)
Comment (REDNBLACK) [reply modules/akkaHttp/src/main/scala/tschema/akkaHttp/FindKey.scala]: Maybe just write a scalafix rule?

**Comment by REDNBLACK** [comment]
@danslapman we gain better tooling support for non-IDE users, less typing == delayed CT syndrome. And, honestly, running scalafix command is not such a trouble (Compare that to deprecated Symbol literal in Scala 2.13)

File: build.sbt (hunk 1 above)
Comment (REDNBLACK) [reply build.sbt]: @Odomontois Quick question: Do u prefer symbol literals to strings?

**Comment by danslapman** [comment]
@REDNBLACK if end-users have transitive dependencies with typed-schema extensions, they will need to rebuild their ecosystem. Honestly I can't remember when I saw 

> non-IDE users

last time in the wild

**Pull request closed without merging**